momentGW/pbc/ints.py

```diff
@@ -679,5 +683,24 @@ def naux(self):
             if self._naux is not None:
                 return self._naux
             else:
-                return [self.naux_full] * len(self.kpts)
-        return [c.shape[-1] if c is not None else self.naux_full for c in self._rot]
+                return self.naux_full
+
+        return [
+            c.shape[-1] if c is not None else self.naux_full[i] for i, c in enumerate(self._rot)
+        ]
+
+    @property
+    def naux_full(self):
+        """
```

Comment:
actually can you change "Return the ..." to "Get the ..." here, in line with my docstring changes? I know, serious nitpicking....